startPage.js

```diff
@@ -0,0 +1,48 @@
+let gameStory = " Sometimes it can be hard to balance our life. This game will give you the possibility of      playing with different aspects that we considered important in order to have a good           balance. There are 4 sections and they will add up depending on the items you grab.        Once you achieve 10 items of each section the game will be won but... be  carefull since   there are robots that will kill you once you collide against them and there is only one         question to be answered here... Are you ready to start gaming with your life? "
```

Comment:
the naming of the file makes no sense to me .. should probably be called "Text.js" and only contain the "Text" class. the `gameStory` variable should be moved to another file (the main file)